tests/test_chromatic.py

```diff
@@ -2199,6 +2199,8 @@ def test_phot():
                           # OpticalPSF.  We effectively rotate the PSF 180 degrees here, but not the
                           # galaxy, so we end up with a slightly different profile.  I (JM) think we
                           # just got lucky before.
+                          # Bumped again.  We were barely failing with 5.0361e-4 > 5e-4 after adding
+                          # the time sampling.
```

Comment:
I'm not sure we need the complete history of this atol choice.  We rarely have a priori-derived tolerances.  Very often these are just something that seems suitably small and passes.  Honestly, I think just writing "OpitcalPSF doesn't match quite as well as the others." is completely sufficient as justification.